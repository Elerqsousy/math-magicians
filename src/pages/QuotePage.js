import React from 'react';

const QuotePage = () => (
  <div className="pageContainer quotePage">
    <h2>
      Mathematics is not about numbers, equations, computations, or algorithms:
      it is about understanding. --Williams Paul Thruston
    </h2>
  </div>
);

export default QuotePage;
