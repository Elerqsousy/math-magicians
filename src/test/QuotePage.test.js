import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import QuotePage from '../pages/QuotePage';
/* eslint-disable no-unused-expressions */

test('navbar to have active links', () => {
  render(<QuotePage />);
  expect(
    screen.getByRole('heading', {
      name: 'Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. --Williams Paul Thruston',
    }),
  ).not.toBeNull;
});

it('Match the Quote component snapshot', () => {
  const tree = renderer.create(<QuotePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
