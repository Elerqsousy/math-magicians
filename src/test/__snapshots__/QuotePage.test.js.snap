// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match the Quote component snapshot 1`] = `
<div>
  <h2>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. --Williams Paul Thruston
  </h2>
</div>
`;
